### ISSUE TITLE
Is there “add to dict" method?

### ISSUE THREAD

**Issue by xaljer** [iamcco/coc-spell-checker#3]

**Comment by iamcco** [comment]
There only add to global user settings or local settings commands by now.

https://github.com/iamcco/coc-spell-checker/blob/master/package.json#L30

**Comment by alex-popov-tech** [comment]
@iamcco hello! is that possible to configure what 'user' dict is? I would like to have separate txt file with all my words there, and not in coc-settings.json...

**Comment by iamcco** [comment]
@alex-popov-tech There is no such command to add word to custom dict yet. But you can use your own dict https://github.com/iamcco/coc-spell-checker#how-to-add-your-own-dictionaries.

**Comment by alex-popov-tech** [comment]
yes...but that not really usable for day-to-day updating dict use case

**Comment by iamcco** [comment]
I don't have time to support it by now. pr is welcome.
